pkg/agent/util/net_windows.go

```diff
@@ -1296,3 +1290,16 @@ func adapterAddresses() ([]*windows.IpAdapterAddresses, error) {
 	}
 	return aas, nil
 }
+
+// ipNetEqual returns true iff both IPNet are equal
+func ipNetEqual(ipn1 *net.IPNet, ipn2 *net.IPNet) bool {
```

Comment:
can we use `IPNetEqual` defined in `pkg/util/ip/ip.go`?

Reply:
We can use `IPNetEqual` in `pkg/util/ip/ip.go` in this PR, but in #5159, we might still need this.

Reply:
Sorry, maybe I am missing some context. Aren't the 2 supposed to do the same thing?

Reply:
I'll verify that. I remember that when I made some tests in #5159,  `IPNetEqual` in `pkg/util/ip/ip.go` cannot be called by the unit tests for Windows code.